package.json

```diff
@@ -84,111 +84,111 @@
+    "punycode": "~2.1.0",
     "qr-image": "3.2.0",
     "random-lib": "2.1.0",
-    "react": "~15.6.1",
-    "react-dnd": "^2.1.4",
-    "react-dnd-html5-backend": "^2.1.2",
-    "react-dom": "^15.5.4",
-    "string.prototype.endswith": "^0.2.0",
-    "string.prototype.startswith": "^0.2.0",
+    "react": "~15.6.2",
+    "react-dnd": "~2.5.1",
+    "react-dnd-html5-backend": "~2.5.1",
+    "react-dom": "~15.6.2",
+    "string.prototype.endswith": "~0.2.0",
+    "string.prototype.startswith": "~0.2.0",
     "tablesort": "5.0.1",
     "tldjs": "1.6.2",
-    "tracking-protection": "1.1.x",
+    "tracking-protection": "~1.1.6",
     "underscore": "1.8.3",
     "url-loader": "~0.6.2",
-    "webtorrent-remote": "^2.0.2"
+    "webtorrent-remote": "~2.0.2"
   },
   "devDependencies": {
     "asar": "~0.13.0",
-    "babel": "^6.1.18",
-    "babel-core": "^6.3.15",
-    "babel-loader": "^7.1.1",
-    "babel-plugin-transform-react-constant-elements": "^6.4.0",
-    "babel-plugin-transform-react-inline-elements": "^6.4.0",
-    "babel-polyfill": "^6.3.14",
-    "babel-preset-env": "^1.1.6",
-    "babel-preset-react": "^6.1.18",
-    "babel-register": "^6.3.13",
-    "babel-runtime": "^6.3.13",
-    "base64-js": "^1.2.0",
-    "chai": "^3.4.1",
-    "chai-as-promised": "^5.1.0",
-    "co-mocha": "^1.1.2",
-    "cross-env": "^3.1.4",
+    "babel": "~6.23.0",
+    "babel-core": "~6.26.0",
+    "babel-loader": "~7.1.2",
+    "babel-plugin-transform-react-constant-elements": "~6.23.0",
+    "babel-plugin-transform-react-inline-elements": "~6.22.0",
+    "babel-polyfill": "~6.26.0",
+    "babel-preset-env": "~1.6.0",
+    "babel-preset-react": "~6.24.1",
+    "babel-register": "~6.26.0",
+    "babel-runtime": "~6.26.0",
+    "base64-js": "~1.2.1",
+    "chai": "~3.5.0",
+    "chai-as-promised": "~5.3.0",
+    "co-mocha": "~1.2.0",
+    "cross-env": "~3.2.4",
     "css-loader": "~0.28.7",
-    "electron-builder": "^17.1.1",
-    "electron-chromedriver": "^1.7.1",
+    "electron-builder": "~17.10.0",
+    "electron-chromedriver": "~1.7.1",
     "electron-packager": "brave/electron-packager",
     "electron-prebuilt": "brave/electron-prebuilt",
     "empty-port": "0.0.2",
-    "enzyme": "^2.9.1",
+    "enzyme": "~2.9.1",
     "flow-bin": "~0.53.1",
-    "git-rev-sync": "^1.8.0",
-    "gulp": "^3.9.0",
-    "istanbul": "^1.1.0-alpha.1",
-    "joi": "^10.2.2",
+    "git-rev-sync": "~1.9.1",
+    "gulp": "~3.9.1",
+    "istanbul": "~1.1.0-alpha.1",
+    "joi": "~10.6.0",
     "jsdom": "~11.2.0",
     "jsdom-global": "~3.0.2",
-    "json-loader": "^0.5.4",
-    "jsonfile": "^2.2.3",
-    "less": "^2.5.3",
-    "less-loader": "^2.2.1",
-    "level": "^1.7.0",
-    "mkdirp": "^0.5.1",
-    "mocha": "^2.3.4",
-    "mockery": "^2.1.0",
-    "muon-winstaller": "^2.5.5",
-    "ncp": "^2.0.0",
-    "node-gyp": "^3.3.1",
+    "json-loader": "~0.5.7",
+    "jsonfile": "~2.4.0",
+    "less": "~2.7.2",
+    "less-loader": "~2.2.3",
+    "level": "~1.7.0",
+    "mkdirp": "~0.5.1",
```

Comment:
`mkdirp@0.5.1` to https://gist.github.com/luixxiul/1cf8689da05e509d45176588730e5a16#file-npm-txt-L1916